features/creating_order/creating_order_with_different_billing_address.feature

```diff
@@ -17,6 +17,8 @@ Feature: Creating order with different billing address
     Scenario: Creating an order with both addresses specified
         When I create a new order for "jon.snow@the-wall.com" and channel "United States"
         And I add "Stark Coat" to this order
+        And I should not see the address book shipping address select
+        And I should not see the address book billing address select
```

Comment:
Not needed as well. If there's some kind of "wait" we should move it to the specify step.